EpiAware/src/EpiLatentModels/manipulators/CombineLatentModels.jl

```diff
@@ -31,13 +31,11 @@ latent_model()
             P <: AbstractVector{<:String}}
         @assert length(models)>1 "At least two models are required"
         @assert length(models)==length(prefixes) "The number of models and prefixes must be equal"
-        for i in eachindex(models)
-            if (prefixes[i] != "")
-                models[i] = PrefixLatentModel(models[i], prefixes[i])
-            end
-        end
+        prefix_models = [prefixes[i] == "" ? models[i] :
```

Comment:
Pedantic point, but this is array comprehension rather than `map`. There isn't a difference in output but array comprehension guarantees a sequence of operations in array construction (this is used in e.g. `Flux` recurrent NNs), whereas `map` doesn't. 

As of Julia > `0.5`, `map` is a sometimes a (small) bit faster than array comprehension (e.g [this discussion](https://discourse.julialang.org/t/map-vs-loops-array-comprehensions-in-julia-1-0/16137)). However, in this case its probably a judgement call on code readability which one to use.

Reply:
Yeah fair enough. I started with map and ported over as I felt like this was easier to read. I don't feel strongly.

Reply:
Going to resolve but could maybe have an issue to use a standard iterator pattern everywhere